coalahtml/coala_html.py

```diff
@@ -92,7 +92,9 @@ def main():
                       sort_keys=True,
                       indent=2,
                       separators=(',', ': '))
-    if not args.nolaunch:
+    if args.nolaunch:
+        sys.exit(exitcode)
```

Comment:
Instead of `exitcode` you should use a specific value or maybe output a message.